docs/tasks/access-kubernetes-api/migrate-third-party-resource.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Migrate a ThirdPartyResource to CustomResourceDefinition
+assignees:
+- enisoc
+- deads2k
+---
+
+{% capture overview %}
+This page shows how to migrate data stored in a ThirdPartyResource (TPR) to a CustomResourceDefinition (CRD).
+
+Kubernetes does not automatically migrate existing TPRs because of API changes introduced as part of
```

Comment:
Break up for clarity:

*Kubernetes does not automatically migrate existing TPRs*. This is due to API changes...